lib/recurly/adyen.js

```diff
@@ -0,0 +1,53 @@
+import Emitter from 'component-emitter';
+import errors from '../errors';
+import {Pricing} from './pricing';
+import {normalize} from '../util/normalize';
+
+const debug = require('debug')('recurly:adyen');
+
+/**
+ * Instantiation factory
+ *
+ * @param  {Object} options
+ * @return {Adyen}
+ */
+export function factory (options) {
+  return new Adyen(Object.assign({}, options, { recurly: this }));
+};
+
+
+/**
+ * Initializes an Adyen session.
+ *
+ * @param {Object} options
+ * @constructor
+ * @public
+ */
+
+class Adyen extends Emitter {
+  constructor (options) {
+    super();
+
+    this._ready = false;
```

Comment:
`_ready` is never checked. Should it be on start?

Reply:
not sure its necessary, going to remove